jerry-libc/jerry-libc.c

```diff
@@ -158,41 +158,15 @@ CALL_PRAGMA (GCC diagnostic pop)
 int
 strcmp (const char *s1, const char *s2)
```

Comment:
This version is less efficient than Xin's implementation (it has an additional compare).